src/web/modules/gateway_accounts/stripe_statement_descriptor.njk

```diff
@@ -0,0 +1,29 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% extends "layout/layout.njk" %}
+
+{% block main %}
+  <span class="govuk-caption-m">Update gateway account</span>
+  <h1 class="govuk-heading-m">Stripe statement descriptor</h1>
+
+  <div>
+    <a href="/gateway_accounts/{{ account.gateway_account_id }}" class="govuk-back-link">Gateway account ({{ account.gateway_account_id }})</a>
+  </div>
+
+  <form method="POST" action="/gateway_accounts/{{ account.gateway_account_id }}/stripe_statement_descriptor">
+    {{ govukInput({
+      id: "statement_descriptor",
+      name: "statement_descriptor",
+      label: { text: "Stripe statement descriptor" },
+      autocomplete: "off"
+    })
+    }}
```

Comment:
Might be good to put the rules for statement descriptor here (maximum length and allowed characters) because no-one can ever remember them. Could even add validation based on that if you’re feeling bold.

Reply:
👍 great ideas, have added a lot of context for what statement descriptors are in https://github.com/alphagov/pay-toolbox/pull/332/commits/f18aed2c17d9968092b91a98471a35a4f751db47 

If Stripe rejects this request the reason why will be shown to the user in an error message, I think we can rely on this for now, added benefit of if they change their criteria for any reason we won't have to keep up to date with a moving target.